pom.xml

```diff
@@ -247,13 +247,13 @@
     </dependencies>
 
     <properties>
-        <gravitee-bom.version>2.5</gravitee-bom.version>
+        <gravitee-bom.version>2.7</gravitee-bom.version>
         <gravitee-common.version>1.25.0</gravitee-common.version>
         <gravitee-plugin.version>1.23.1</gravitee-plugin.version>
         <gravitee-expression-language.version>1.9.2</gravitee-expression-language.version>
         <gravitee-reporter-api.version>1.22.0</gravitee-reporter-api.version>
         <gravitee-tracing-api.version>1.0.0</gravitee-tracing-api.version>
-        <gravitee-kubernetes.version>0.4.0</gravitee-kubernetes.version>
+        <gravitee-kubernetes.version>1.0.0-SNAPSHOT</gravitee-kubernetes.version>
```

Comment:
Are you sure about it ? Don't see the link with rxjava3.

Reply:
yep, gravitee-kubernetes has been migrated to rxJava3.
I'll change the version when it will be merged (it also requires gravitee-node)